jaeger/cmd/install.go

```diff
@@ -68,27 +67,29 @@ func newCmdInstall() *cobra.Command {
 		`Skip checks for namespace existence`,
 	)
 
-	// TODO: Add --set flag set and also config
+	flags.AddValueOptionsFlags(cmd.Flags(), &options)
 
 	return cmd
 }
 
-func install(w io.Writer, values *jaeger.Values) error {
+func install(w io.Writer, options values.Options) error {
+
+	// Create final values override
```

Comment:
I'd omit the word "final" here.  This is confusing because I'm not sure what is "final" about the overrides.  (they still need to be applied on top of the defaults)